### DISCUSSION TITLE
cannot find module "axios"

### DISCUSSION THREAD

**Discussion by ghost** [ASEAN-Build-The-Earth/Merlion#46] [Q&A] [answered]
So, i done everything correctly and i am stuck at the point where you turn the bot on. The thing is that it said required axios module. What should i do from here. And im sending a screenie to show you what i mean 
![image](https://user-images.githubusercontent.com/54899587/137144945-506161ed-8e37-4af7-accd-20d4e8bdb071.png)

**Comment by tintinkung** [comment, marked as answer]
type
```
npm ci
```
on your terminal console

**Comment by ghost** [comment]
ok thanks
